### PR TITLE
Fix annoying typing issues

### PR DESCRIPTION
**Pull request by FalseLobster**
Apparently we just need to explicitly declare which test framework types we wanna use... 🤦‍♀️

### CODE REVIEW

File: scripts/tasks/ts.js [hunk 1]
```diff
@@ -2,11 +2,18 @@
 
 const path = require('path');
 const { tscTask, argv } = require('just-scripts');
+const { readdirSync } = require('fs');
```
Comment (dotnetjunky) [inline scripts/tasks/ts.js:5]: why is this added?

File: packages/components/Button/src/Button.test.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { Button } from './Button';
 import * as renderer from 'react-test-renderer';
+import '@types/jest';
```
Comment (dotnetjunky) [inline packages/components/Button/src/Button.test.tsx:4]: Can't you achieve the same by updating the `types` attribute in the `tsconfig.json` file?

```ts
compilerOptions: {
    types: ["node", "jest"]
}
```
Then you don't have to do this in every test file.

Comment (ksiler) [reply packages/components/Button/src/Button.test.tsx:4]: We thought about this but we'd need to list every other type it requires too, right? That'd be a long list. All to ensure 'jasmine' doesn't get referenced.

What would be nice is an attribute that does the opposite of this but I don't believe it exists.

Comment (dotnetjunky) [reply packages/components/Button/src/Button.test.tsx:4]: Have you actually tried it? I'd think this `types` attribute only affect global type declaration. If jest explicitly include other modules, those types will be checked by Typescript.

Comment (ksiler) [reply packages/components/Button/src/Button.test.tsx:4]: I see. Yeah that works for me locally.

File: scripts/tasks/ts.js (hunk 1 above)
Comment (FalseLobster) [reply scripts/tasks/ts.js:5]: Left overs from the first commit. You can see I was programmatically trying to enumerate all the @types except Jasmine and pass it in the commandline to tsc
